targets/TARGET_STM/TARGET_STM32H7/TARGET_STM32H747xI/TARGET_PORTENTA_H7/COMPONENT_WHD/mbed_lib.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "cy_psoc6_whd",
```

Comment:
Is it the right place to add this COMPONENT ?
Maybe could be better in connectivity directory ?

Reply:
Making the whole WHD driver completely HW independent is not a super easy task (and needs coordination with Cypress folks). In the meantime I preferred to keep it internal so it's clear that it needs further rework to be used with other boards)